src/core/Akka/Event/ILoggingAdapter.cs

```diff
@@ -46,10 +44,6 @@ public interface ILoggingAdapter
         /// <param name="args">The arguments.</param>
         void Warning(string format, params object[] args);
 
-        [Obsolete("Use Warning instead!")]
```

Comment:
No, I meant that this is a good idea. Either you end up leaving obsoleted stuff around permanently or you tear the band-aid off right away. I'm a fan of tearing the band-aid off.